src/driver/safe/alloc.rs

```diff
@@ -18,6 +18,10 @@ pub unsafe trait DeviceRepr {
     fn as_kernel_param(&self) -> *mut std::ffi::c_void {
         self as *const Self as *mut _
     }
+    #[inline(always)]
+    fn maybe_stream(&self) -> Option<&'_ CudaStream> {
+        None
```

Comment:
Not a breaking change since there's a default impl, but not quite sure if we need this. This would be used in CudaStream::launch/CudaStream::launch_cooperative to check if the args belong to the launching stream